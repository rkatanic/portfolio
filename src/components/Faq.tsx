import Accordion from "./Accordion";
import "./Faq.css";
import Section from "./Section";

const Faq = (): JSX.Element => (
  <Section id="faq">
    <h3 className="headline">Frequently Asked Questions</h3>
    <Accordion />
  </Section>
);

export default Faq;
