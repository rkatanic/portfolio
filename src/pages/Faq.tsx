import Accordion from "../components/Accordion";
import Section from "../components/Section";

import "./Faq.css";

const Faq = (): JSX.Element => (
  <Section id="faq">
    <h3 className="headline">Frequently Asked Questions</h3>
    <Accordion />
  </Section>
);

export default Faq;
