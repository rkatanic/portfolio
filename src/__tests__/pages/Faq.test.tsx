import { render } from "@testing-library/react";
import Faq from "../../pages/Faq";

describe("FAQ page", (): void => {
  it("should render", (): void => {
    const { baseElement } = render(<Faq />);

    expect(baseElement).toMatchSnapshot();
  });
});
