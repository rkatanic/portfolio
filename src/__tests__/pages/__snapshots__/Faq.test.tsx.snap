// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`FAQ page should render 1`] = `
<body>
  <div>
    <section
      class="section"
      id="faq"
    >
      <h3
        class="headline"
      >
        Frequently Asked Questions
      </h3>
      <div
        class="accordion"
      >
        <li
          class="accordion-item accordion-item-active"
        >
          <div
            class="accordion-item-label"
          >
            <p
              class="accordion-item-label-headline"
            >
              What is current tech stack you are working with
              <span
                class="primary"
              >
                ?
              </span>
            </p>
            <svg>
              arrow-up.svg
            </svg>
          </div>
          <p
            class="accordion-item-text accordion-item-text-open"
          >
            Currently I'm mostly working with React.js on front-end and Spring Boot framework on back-end.
            <br />
            <br />
          </p>
        </li>
        <li
          class="accordion-item "
        >
          <div
            class="accordion-item-label"
          >
            <p
              class="accordion-item-label-headline"
            >
              What did you use to build your website
              <span
                class="primary"
              >
                ?
              </span>
            </p>
            <svg>
              arrow-up.svg
            </svg>
          </div>
          <p
            class="accordion-item-text "
          >
            This website is built with React.js. If you are interested in source code, you can find it here.
            <br />
            <br />
          </p>
        </li>
        <li
          class="accordion-item "
        >
          <div
            class="accordion-item-label"
          >
            <p
              class="accordion-item-label-headline"
            >
              I have a project for you, are you available
              <span
                class="primary"
              >
                ?
              </span>
            </p>
            <svg>
              arrow-up.svg
            </svg>
          </div>
          <p
            class="accordion-item-text "
          >
            At this moment I'm full time employed. I could offer a few hours of my services per week , if this suits you.
            <br />
            <br />
          </p>
        </li>
        <li
          class="accordion-item "
        >
          <div
            class="accordion-item-label"
          >
            <p
              class="accordion-item-label-headline"
            >
              How to pronounce your name correctly
              <span
                class="primary"
              >
                ?
              </span>
            </p>
            <svg>
              arrow-up.svg
            </svg>
          </div>
          <p
            class="accordion-item-text "
          >
            When pronounced correctly my first name sounds something like ra-dee-woh-ye.
            <br />
            <br />
          </p>
        </li>
      </div>
    </section>
  </div>
</body>
`;
